index.js

```diff
@@ -107,6 +108,7 @@ LFU.prototype.remove = function(key) {
 };
 
 LFU.prototype.removeFromParent = function(parent, key) {
+	Object.setPrototypeOf(parent.items, new Set());
```

Comment:
parent.items is always a set in the current implementation, so casting it into a set again should have no effect on the existing functionality, however, when import is used, since we already know that parent.items is a set, it is cast again. I am already using the modified library in one of my projects.